sdk/core/core-tracing/samples-dev/basicTracing.ts

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Uses `@azure/core-tracing` APIs to instrument method calls for a fake Azure client library.
+ */
+
+import { createTracingClient, OperationTracingOptions, TracingClient } from "@azure/core-tracing";
+
+/**
+ * Represents common operation options
+ */
+interface OperationOptions {
+  tracingOptions?: OperationTracingOptions;
+}
+
+/**
+ * Represents an Azure Client that does work against an Azure service.
+ */
+class BasicClient {
+  private tracingClient: TracingClient;
+
+  constructor() {
+    // Create a tracing client in the constructor, passing to it
+    // the package name, the value for az.namespace attribute, and the
+    // optional package version.
+    this.tracingClient = createTracingClient({
+      packageName: "@azure/core-tracing",
```

Comment:
I wonder if we should make this something like `@azure/some-example-package` to make it clear you don't pass `@azure/core-tracing` to itself

Reply:
Good idea, fixed (and added the link per your other comment) 👍